src/component/history/EventHistory/EventHistory.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
```

Comment:
Seems like a candidate for tsx? New file and all?

Reply:
Certainly.

Reply:
Certainly. But I noticed when I started converting App.tsx to type

> Not showing "placeholder" for feature toggles when signing in:
> 
> https://www.loom.com/share/5780b2497f9241de9518eb874a56e0c4

Thanks. Handling that by resetting loading count on logout now.